CoEpi/ui/HealthQuiz/HealthQuizViewController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  HealthQuizViewController.swift
+//  CoEpi
+//
+//  Created by Dusko Ojdanic on 3/26/20.
+//  Copyright © 2020 org.coepi. All rights reserved.
+//
+
+import UIKit
+
+class HealthQuizViewController: UIViewController {
+    private let viewModel: HealthQuizViewModel
+
+    @IBOutlet weak var questionList: UITableView!
+    
+    init(viewModel: HealthQuizViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: String(describing: Self.self), bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    @IBAction func submit(_ sender: UIButton) {
+        viewModel.onTapSubmit()
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        questionList.register(QuestionCell.self, forCellReuseIdentifier: "QuestionCell")
+        questionList.delegate = self
+        questionList.dataSource = self
+    }
+}
+
+extension HealthQuizViewController: UITableViewDataSource, UITableViewDelegate {
```

Comment:
It's slightly cleaner to implement `UITableViewDataSource` as a separate class. See `DebugViewController`

Reply:
Removed `UITableViewDelegate`, dont actually need it. Not sure it's much cleaner since both the VC and datasource need the view model, but done.